doc/stdlib/collections.rst

```diff
@@ -221,6 +221,12 @@ Iterable Collections
 
    Returns an array containing only the unique elements of the iterable ``itr``\ , in the order that the first of each set of equivalent elements originally appears. If ``dim`` is specified, returns unique regions of the array ``itr`` along ``dim``\ .
 
+   .. code-block:: julia
+
+       unique(f, itr)
+
+   Returns an array containing one value from ``itr`` for each unique value produced by ``f`` applied to elements of ``itr``\ .
+
```

Comment:
(note that there are several different signatures, and the one in `set.jl` that takes an unspecified iterable is different from `unique(::AbstractArray, dim::Int)` definition in `multi.jl`. I tried a variety of different combinations, including splitting the doc strings up)